buildconfig/CMake/FindBoostPython.cmake

```diff
@@ -5,7 +5,11 @@
 #  - windows: boost_python (python2), ????? (python3)
 #  - others?
 if ( MSVC )
-  find_package ( Boost COMPONENTS python REQUIRED )
+  if ( ${Boost_VERSION} LESS 106700)
```

Comment:
Maybe we don't need the `if` check? We could assume that on windows we always know what version of boost we're running.

Reply:
Good point